packages/grafana-ui/src/components/Table/TableNG/utils.ts

```diff
@@ -84,6 +84,25 @@ export function shouldTextWrap(field: Field): boolean {
   return Boolean(field.config.custom?.wrapText);
 }
 
+export function getMaxHeight(field: Field): number | undefined {
+  return field.config?.custom?.maxHeight;
```

Comment:
maybe have this always return a number so you dont need to check the return value with `typeof maxHeight 'number'`?

instead of undefined, do `?? Infinity`

Reply:
we would just need to check whether it was `Infinity` in that case though, I feel like the undefined is equally clear. If you're concerned about the `typeof`, I'll just change it to `maxHeight != null`